docs/install.rst

```diff
@@ -11,20 +11,15 @@ pypromice can installed using pip:
 Developer install
 *****************
 
-pypromice can be ran in an environment with the specified dependencies.
+pypromice can be ran in an environment with the pypromice package cloned from GitHub_. 
 
 .. code:: console
 
 	$ conda create --name pypromice python=3.8
 	$ conda activate pypromice
-	$ conda install xarray pandas pathlib
-	$ conda install -c conda-forge netCDF4
```

Comment:
Not quite clear to me why we are removing the recommendation to install these packages? Is the `requirements.txt` sufficient? (applies to `scikit-learn` below as well)

Reply:
I removed them because there was confusion about mixing conda and pip installations, and there was problems with reproducibility. By pip installing pypromice, the dependencies are automatically installed. 

If we set up a pip package and a conda package then we can modify this to reflect a conda install guide alongside a pip install guide.